src/tests/functional/plugin/gpu/behavior/cache.cpp

```diff
@@ -1,119 +0,0 @@
-// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
is this test only used by gpu?, I see the code is moved to "src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp" in this PR

Reply:
recommend to add a comment in code if only used for GPU, but in plugin shared folder

Reply:
the case has been generalized and can be used by whatever plugin if want to cover some basic cache test.